<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>718</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="picture.qrc">
    <normaloff>:/png/soft.png</normaloff>:/png/soft.png</iconset>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>3</y>
      <width>151</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>串口设置</string>
    </property>
    <widget class="QPushButton" name="closeSerial">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>180</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭串口</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openSerial">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>打开串口</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>125</width>
       <height>152</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>串行口：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="portName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>波特率：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="baudRate"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>数据位：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="dataBits"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>校验位：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="parityCom"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>停止位：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="stopBit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>数据流：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="flowType"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>1011</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>496</y>
      <width>171</width>
      <height>185</height>
     </rect>
    </property>
    <property name="title">
     <string>数据提取</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="filetrans_select">
         <property name="text">
          <string>选择数据记录文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="filetrans_select_display"/>
       </item>
       <item>
        <widget class="QPushButton" name="dir_datasave_select">
         <property name="text">
          <string>选择数据导出目录</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="dir_datasave_select_display"/>
       </item>
       <item>
        <widget class="QPushButton" name="filetrans_start">
         <property name="text">
          <string>提取数据</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="filetrans_progressBar">
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="toolTipDuration">
          <number>-1</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>570</y>
      <width>1001</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>实时数据</string>
    </property>
    <widget class="QLineEdit" name="magnetism2all_display">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>80</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="acc_y_display">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>40</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="magnetism2x_display">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="magnetism1all_display">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>80</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="acc_z">
     <property name="geometry">
      <rect>
       <x>782</x>
       <y>61</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>加计Z：</string>
     </property>
    </widget>
    <widget class="QLabel" name="timerec">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>21</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>时间：</string>
     </property>
    </widget>
    <widget class="QLabel" name="pump01">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>81</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>光泵：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="height_display">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="acc_z_display">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>60</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="magnetism2all">
     <property name="geometry">
      <rect>
       <x>632</x>
       <y>81</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>磁通门2A：</string>
     </property>
    </widget>
    <widget class="QLabel" name="latitude">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>42</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>纬度：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="roll_display">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>40</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="acc_x">
     <property name="geometry">
      <rect>
       <x>782</x>
       <y>22</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>加计X：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="magnetism1z_display">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>60</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="magnetism1y">
     <property name="geometry">
      <rect>
       <x>482</x>
       <y>42</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>磁通门1Y：</string>
     </property>
    </widget>
    <widget class="QLabel" name="pos_status">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>61</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>定位状态：</string>
     </property>
    </widget>
    <widget class="QLabel" name="magnetism1all">
     <property name="geometry">
      <rect>
       <x>482</x>
       <y>81</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>磁通门1A：</string>
     </property>
    </widget>
    <widget class="QLabel" name="magnetism2x">
     <property name="geometry">
      <rect>
       <x>632</x>
       <y>22</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>磁通门2X：</string>
     </property>
    </widget>
    <widget class="QLabel" name="yaw">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>61</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>航向：</string>
     </property>
    </widget>
    <widget class="QLabel" name="magnetism2y">
     <property name="geometry">
      <rect>
       <x>632</x>
       <y>43</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>磁通门2Y：</string>
     </property>
    </widget>
    <widget class="QLabel" name="acc_y">
     <property name="geometry">
      <rect>
       <x>782</x>
       <y>42</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>加计Y：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="magnetism2z_display">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>60</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="pitch">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>22</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>俯仰：</string>
     </property>
    </widget>
    <widget class="QLabel" name="longitude">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>22</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>经度：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pitch_display">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="acc_x_display">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>20</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="height">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>61</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>高程：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pos_status_display">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>108</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="pump_display">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>80</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="timerec_display">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>108</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="roll">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>42</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>横滚：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="longti_display">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="magnetism1z">
     <property name="geometry">
      <rect>
       <x>482</x>
       <y>61</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>磁通门1Z：</string>
     </property>
    </widget>
    <widget class="QLabel" name="ins_status">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>40</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>组导状态：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="magnetism1y_display">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>40</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="magnetism2y_display">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>40</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="ins_status_display">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>108</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="magnetism2z">
     <property name="geometry">
      <rect>
       <x>632</x>
       <y>62</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>磁通门2Z：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="yaw_display">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>60</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lati_display">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="magnetism1x_display">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="magnetism1x">
     <property name="geometry">
      <rect>
       <x>482</x>
       <y>22</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>磁通门1X：</string>
     </property>
    </widget>
    <widget class="QLabel" name="acc_status">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>81</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>加计状态：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="acc_status_display">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>108</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="risefall_display">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="risefall">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>起伏：</string>
     </property>
    </widget>
    <widget class="QLabel" name="MCU_status">
     <property name="geometry">
      <rect>
       <x>783</x>
       <y>81</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>数采：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="MCUsave_status_display">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>80</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>215</y>
      <width>171</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>UDP组播设置</string>
    </property>
    <widget class="QPushButton" name="UDPshutdown">
     <property name="geometry">
      <rect>
       <x>82</x>
       <y>95</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
    <widget class="QPushButton" name="UDPconnect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>95</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>加入</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="UDPsendButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="UDPrecvButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>接收</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="multicastIP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>43</y>
       <width>48</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>组播IP：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="multicastIPlineedit">
     <property name="geometry">
      <rect>
       <x>67</x>
       <y>43</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>227.0.81.22</string>
     </property>
    </widget>
    <widget class="QLabel" name="localport">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>接收端口：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="localportlineedit">
     <property name="geometry">
      <rect>
       <x>67</x>
       <y>70</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>37001</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>342</y>
      <width>181</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>绘图控制</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>128</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QCheckBox" name="fluxgate1y">
        <property name="text">
         <string>磁通门Y1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="fluxgate2z">
        <property name="text">
         <string>磁通门Z2</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="pump1">
        <property name="text">
         <string>光泵1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="fluxgate2all">
        <property name="text">
         <string>磁通门2总场</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="fluxgate1x">
        <property name="text">
         <string>磁通门X1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="fluxgate2x">
        <property name="text">
         <string>磁通门X2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="fluxgate2y">
        <property name="text">
         <string>磁通门Y2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="fluxgate1z">
        <property name="text">
         <string>磁通门Z1</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="pump2">
        <property name="text">
         <string>光泵2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="fluxgate1all">
        <property name="text">
         <string>磁通门1总场</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>groupBox_5</zorder>
   <zorder>groupBox_6</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="picture.qrc"/>
 </resources>
 <connections/>
</ui>
